app/models/provider.rb

```diff
@@ -76,4 +76,8 @@ def lacks_admin_users?
   def geocoded?
     latitude.present? && longitude.present?
   end
+
+  def uniq_sites
+    sites.uniq(&:code)
```

Comment:
if we are showing provider sites across different cycles, and only show them per uniq code, does that mean that we might miss any changes in the address across the cycles and not surface them?